index.bs

```diff
@@ -630,6 +633,9 @@ with the following additional codes:
   <dt><dfn for=errors export>no such user context</dfn>
   <dd>Tried to reference an unknown [=user context=].
 
+  <dt><dfn for=errors export>no such web extension</dfn>
+  <dd>Tried to interact with not installed web extension.
```

Comment:
Maybe we can change the error message so that it can more easily be re-used for other commands later as well?

```suggestion
  <dd>Tried to reference an unknown web extension.
```